esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -660,7 +757,12 @@ class GridToMeshESMFRegridder:
     r"""Regridder class for rectilinear to unstructured :class:`~iris.cube.Cube`\\ s."""
 
     def __init__(
-        self, src_mesh_cube, target_grid_cube, mdtol=1, precomputed_weights=None
+        self,
+        src_mesh_cube,
+        target_grid_cube,
+        mdtol=None,
```

Comment:
Also need to change the docstring to:

```
mdtol : float, optional
```

... with possible references to the defaults for the different `method` values.